modules/lfs/server.go

```diff
@@ -69,6 +69,11 @@ func (v *RequestVars) ObjectLink() string {
 	return fmt.Sprintf("%s%s/%s/info/lfs/objects/%s", setting.AppURL, v.User, v.Repo, v.Oid)
 }
 
+// VerifyLink builds a URL for verifying the object.
+func (v *RequestVars) VerifyLink() string {
+	return fmt.Sprintf("%s%s/%s/info/lfs/verify", setting.AppURL, v.User, v.Repo)
```

Comment:
nit: use URLJoin. The current code assumes `setting.AppURL` has a trailing slash, and even if that assumption always holds (for now), it'd be nicer to not have to assume it in the first place.